events/wmt20.md

```diff
@@ -217,3 +217,59 @@ Friday, November 20, 2020
 | | [***The Tatoeba Translation Challenge – Realistic Data Sets for Low Resource and Multilingual MT***](https://aclanthology.org/2020.wmt-1.139.pdf) <br>Jörg Tiedemann |
 | | [***Human-Paraphrased References Improve Neural Machine Translation***](https://aclanthology.org/2020.wmt-1.140.pdf) <br>Markus Freitag, George Foster, David Grangier and Colin Cherry |
 | |	[***Incorporating Terminology Constraints in Automatic Post-Editing***](https://aclanthology.org/2020.wmt-1.141.pdf) <br>David Wan, Chris Kedzie, Faisal Ladhak, Marine Carpuat and Kathleen McKeown |
+
+## Results
```

Comment:
We don't really clearly say that these are the "winning"/"best"/"1st-place" systems.

So "Average" etc looks like it could be average score of all systems...

We need to somehow communicate this clearly.

Also making "System" the 2nd column (before its score) would help.

Reply:
Made changes according to your comments, @bittlingmayer. I guess the results are now easier to comprehend as the "System" column changed places with the columns representing the scores. 
I've also excluded the "Human" results from the list and replaced them with first in the line "System" results.